plugin/src/main/java/fr/mrmicky/worldeditselectionvisualizer/WorldEditSelectionVisualizer.java

```diff
@@ -69,6 +72,10 @@ public void onEnable() {
 
         getServer().getOnlinePlayers().forEach(this::loadPlayer);
 
+        if (getConfig().getBoolean("hook-placeholders")) {
```

Comment:
I would prefer to verify that PlaceHolderAPI is enabled rather than using a config option, to prevent a `NoClassDefFoundError` if the option is enabled in the config but PlaceHolderAPI is not installed
```suggestion
        if (getServer().getPluginManager().isPluginEnabled​("PlaceHolderAPI")) {
```